partcad/src/partcad/user_config.py

```diff
@@ -128,5 +128,12 @@ def __init__(self):
         else:
             self.max_script_correction = None
 
+        # option: git.config
+        # description: git configuration for git
+        # values: <dict>
+        # default: {}
+        self.git_config = self.config_obj.get("git", {}).get("config", {})
+        if self.git_config is None:
+            self.git_config = {}
```

Comment:
Please add section in docs with an example, before [Generative AI](https://partcad.readthedocs.io/en/latest/features.html#generative-ai) section, it's in `docs/source/features.rst`. You can use the following:

## Installing Dependencies

Cloning over SSH is faster and more reliable because it uses an efficient protocol with lower overhead, supports compression, and maintains stable connections via key-based authentication. SSH avoids HTTPS rate limits, handles firewalls better, and eliminates credential prompts, making it ideal for large repositories or frequent interactions.

If you have SSH keys configure then you can add the following to the `~/.partcad/config.yaml`:

```yaml
git:
  config:
    url."git@github.com:".insteadOf: "https://github.com/"
```

Reply:
Ok, @alexanderilyin I'll update the docs